_rules/audio-text-alternative-e7aa44.md

```diff
@@ -100,6 +100,22 @@ This `audio` element describes some of the text on the same page. The text on th
 </html>
 ```
 
+#### Passed Example 3
+
+This `audio` element autoplays, and there is a transcript of the audio.
+
+```html
+<html lang="en">
+	<audio src="/test-assets/moon-audio/moon-speech.mp3" autoplay controls></audio>
+	<p>
+		The above audio contains the following speech: We choose to go to the moon in this decade and do the other things,
+		not because they are easy, but because they are hard, because that goal will serve to organize and measure the best
+		of our energies and skills, because that challenge is one that we are willing to accept, one we are unwilling to
+		postpone, and one which we intend to win, and the others, too.
+	</p>
+</html>
+```
+
 ### Failed
```

Comment:
There should probably be a Failed case with an autoplay attribute.

Reply:
Thanks @Jym77. 

I modified Fail Ex 2 to autoplay.